crawler/dockermonitor.py

```diff
@@ -0,0 +1,43 @@
+import sys
+from containerevent import ContainerEvent
+import docker
+
+class DockerMonitor(object):
+        def __init__(self, eventQ):
+                self.eventQ = eventQ
+		self.client = docker.Client(base_url='unix://var/run/docker.sock')
```

Comment:
These two lines are not aligned...  Tabs vs. spaces?  Autopep8 can find and fix.